openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSL.java

```diff
@@ -699,4 +702,68 @@ public static native void setCertificateBio(
      * Free x509 chain ({@code STACK_OF(X509)} pointer).
      */
     public static native void freeX509Chain(long x509Chain);
+    
+    // Extracted from https://github.com/openssl/openssl/blob/master/include/openssl/tls1.h
+    
+    /**
+     * This is the default (initial) status type value for {@link SSLEngine}s.
+     * 
+     * @see #setStatusType(long, int)
+     */
+    public static final int TLSEXT_STATUSTYPE_HOSTNAME = 0;
```

Comment:
we should derive these values from JNI to ensure they remain consistent. See [NativeStaticallyReferencedJniMethods](https://github.com/netty/netty-tcnative/blob/master/openssl-dynamic/src/main/java/io/netty/internal/tcnative/NativeStaticallyReferencedJniMethods.java)